src/page-modules/contact/components/input/description-modal.tsx

```diff
@@ -0,0 +1,94 @@
+import { Typo } from '@atb/components/typography';
+import style from './input.module.css';
+import { motion } from 'framer-motion';
+import { FocusScope } from '@react-aria/focus';
+import { MonoIcon } from '@atb/components/icon';
+import { useEffect, useRef } from 'react';
+
+type DescriptionModalProps = {
+  title: string;
+  description: string;
+  ariaLabel: string;
+  isModalOpen: boolean;
+  closeModal: () => void;
+};
+
+const DescriptionModal = ({
+  title,
+  description,
+  ariaLabel,
+  isModalOpen,
+  closeModal,
+}: DescriptionModalProps) => {
+  const modalRef = useRef<HTMLDivElement | null>(null);
+  const wrapperRef = useRef<HTMLDivElement | null>(null);
+
+  const handleEscape = (event: KeyboardEvent) => {
+    if (event.key === 'Escape') {
+      closeModal();
+    }
+  };
+
+  const handleClickOutside = (event: MouseEvent) => {
+    if (
+      wrapperRef.current &&
+      modalRef.current &&
+      !modalRef.current.contains(event.target as Node)
+    ) {
+      closeModal();
+    }
+  };
+
+  useEffect(() => {
+    document.addEventListener('keydown', handleEscape);
+    if (wrapperRef.current) {
+      wrapperRef.current.addEventListener('click', handleClickOutside);
+    }
+    return () => {
+      document.removeEventListener('keydown', handleEscape);
+      if (wrapperRef.current) {
+        wrapperRef.current.removeEventListener('click', handleClickOutside);
+      }
+    };
+  }, []);
+
+  if (!isModalOpen) return null;
+
+  return (
```

Comment:
Would it be better to use [the dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) for this component? 🤔 At least add the dialog role to the element

Reply:
I also think we need to add aria-labelledby and aria-describedby to help screen readers understand the content.

Reply:
Probably aria-modal too: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-modal